tests/RequestParserTest.php

```diff
@@ -63,6 +63,29 @@ function testRequestViaClosure()
         $this->assertEquals('LIQRGV\QueryFilter\Mocks\MockClosureModel', $builderStruct->baseModelName);
     }
 
+    function testModelSetter()
+    {
+        $uri = 'some_model';
+        $controllerClass = MockModelController::class;
+        $query = new ParameterBag([
+            "sort" => "-name",
+        ]);
+        $requestParserOptions = [
+            'model_namespaces' => [
+                'LIQRGV\QueryFilter\Mocks',
+            ]
+        ];
+
+        $request = $this->createControllerRequest($uri, $controllerClass, $query, $requestParserOptions);
+
+        $requestParser = new RequestParser($request);
+        $requestParser->setModel('LIQRGV\QueryFilter\Mocks\MockModel');
```

Comment:
Can we change this to `MockModel::class`?
I think that's more intuitive than passing string.

Reply:
Oh sure, on next commit :+1:

Reply:
Done